### DISCUSSION TITLE
Feature request: allow resizing images

### DISCUSSION THREAD

**Discussion by dbieber** [athensresearch/athens#850] [Feature Requests]
Feature request: allow resizing images pasted into Athens Research

This idea was mentioned in #25 as requiring datoms and being suitable for implementing later after the initial image support was added (maybe that later is now :)).

I opened issue #849 for this before realizing the guidelines say to open a discussion rather than an issue to start. Sorry about the mixup.

**Comment by dbieber** [comment]
I also see this is a duplicate with #812, which I missed because that one spells resize as re-size, so my search didn't find it.

**Reply by jsmorabito** [reply]
closed #849 and archived my discussion post because yours is more thorough!
